manage-data/migrate/migrate-internal-indices.md

```diff
@@ -7,109 +7,35 @@ applies_to:
   deployment:
     eck: unavailable
     ess: ga
-    ece: unavailable
+    ece: ga
   serverless: unavailable
```

Comment:
same note about `unavailable`

imo, stack flavor is more about availability, deployment is more about applicability